meetings/agenda-minutes-2020-07-09.md

```diff
@@ -0,0 +1,135 @@
+
+    * JGT: That also sounds close to option 2, where the Absolute duration type is a scalar bigint instead of a class.
+
+    * SFC: To elaborate on option 3, a few months ago we had a conversation on how durations relate to calendar systems. Does P1M mean one month in the Gregorian or Hebrew calendar? Basically the duration's meaning is dependent on the context of the date that it's added to. However, we have now seen that this approach can cause software bugs. So, if we are moving away from having durations be defined by their contexts, one other option could be to do option 3 with RFC 5545 durations, but give it a time zone and calendar slot. You still have to interpret it in the context though, because the length of P1M depends on the starting date. That's why we had originally decided not to put the calendar in the duration.
+
+    * PFC: I like USA's idea of using delta nanoseconds for Absolute durations since it is a low-friction way to get this strong typing. I also like the RFC 5545 approach since it is at least unambiguous and although the behaviour may still be surprising it seems like it mostly aligns with programmers' expectations, so hopefully the surprises will be few.
+
+    * SFC: We can also make sure we throw exceptions when doing arithmetic with a type that is unexpected for that kind of duration.
+
+    * JGT: The developer experience of delta nanoseconds does seem worse though. If you want a one-hour warning before the shop closes then you need to turn that hour into nanoseconds.
+
+    * USA: I understand that, but I also think it's valuable for developers to know that they have to convert between these values.
+
+    * JGT: I also think it's difficult to split out the hours from days in a duration like P2DT2H.
+
+    * USA: I think we should have a static method on Duration to convert Duration objects to delta nanoseconds and vice versa.
+
+    * PDL: The only reservation I have is that if we want to keep property bags working, then strong typing is pointless. No-one will use the strong types if a property bag is available.
+
+    * JGT: Unless the property bag is required to have information about the type.
+
+    * PDL: But at that point it runs counter to what we intended. Therefore I think if we go that route we should go fully with AbsoluteDuration.from(propertyBag).
+
+    * RGN: The more I think about it, the more I like paving the cowpath that iCal/RFC 5545 has already established. It's a compelling position to be conveniently compatible with it. If we need to add things to that, then those are the things that can have cumbersome APIs.
+
+    * We will tentatively move forward with RFC 5545 durations.
+
+* [#716](https://github.com/tc39/proposal-temporal/issues/716) Should Absolute.from() have an option to resolve offset vs. timezone conflicts?
+
+* Temporal.parse
+
+* LocalDateTime
+
```

Comment:
I'd delete these lines if we didn't get around to discussing them in the meeting. (At least, that's what I usually do)

Reply:
Done